lib/any.js

```diff
@@ -2,8 +2,20 @@
 
 module.exports = match
 
+match.needsIndex = [
```

Comment:
```suggestion
var needsIndex = [
```

I don’t think this needs to be exported